docs/index.md

```diff
@@ -68,7 +68,7 @@ Configurations for Pace to use NDSL with different backend:
 
   - Run: load pre-compiled program and execute, fail if the .so is not present (_no hash check!_) (backend must be `dace:gpu` or `dace:cpu`)
 
-- PACE_FLOAT_PRECISION=64 control the floating point precision throughout the program.
+- NDSL_FLOAT_PRECISION=64 control the floating point precision throughout the program.
```

Comment:
I'm assuming this controls precision for all things not hard-typed.  For example, I can still hard type physics to use 64-bit and allow this variable to control things at runtime.  If that is correct, we may want to change the description here slightly.

Reply:
Thanks for clarifying and agree we'll fix documentation in the relevant PR (#192)